spec/lib/whitehall_importer/create_edition_spec.rb

```diff
@@ -86,38 +86,148 @@
       expect(edition.editors.count).to eq(2)
     end
 
-    it "sets the timeline history for revision history" do
-      freeze_time do
-        first_user = create(:user)
-        second_user = create(:user)
+    context "has editorial remarks" do
+      let(:first_user) { create(:user) }
+      let(:second_user) { create(:user) }
 
-        user_ids = {
+      let(:user_ids) {
```

Comment:
Typically if the block spans multiple lines we use `do...end` instead of curly braces.

Reply:
No worries - all changed.  Thanks.